test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -27,20 +27,20 @@ images:
     converted: false
     tag: "latest"
 
-  fedora-31:
-    filename: Fedora-Cloud-Base-31-1.9.x86_64.qcow2
-    url: https://download.fedoraproject.org/pub/fedora/linux/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-31-1.9.x86_64.qcow2
-    checksum: sha256:e3c1b309d9203604922d6e255c2c5d098a309c2d46215d8fc026954f3c5c27a0
-    converted: true
-    tag: "latest"
-
   fedora-32:
     filename: Fedora-Cloud-Base-32-1.6.x86_64.qcow2
     url: https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2
     checksum: sha256:423a4ce32fa32c50c11e3d3ff392db97a762533b81bef9d00599de518a7469c8
     converted: true
     tag: "latest"
 
+  fedora-33:
+    filename: Fedora-Cloud-Base-33-1.2.x86_64.qcow2
+    url: https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2
+    checksum: sha256:423a4ce32fa32c50c11e3d3ff392db97a762533b81bef9d00599de518a7469c8
```

Comment:
```
The checksum for /images/base/Fedora-Cloud-Base-33-1.2.x86_64.qcow2 did not match 423a4ce32fa32c50c11e3d3ff392db97a762533b81bef9d00599de518a7469c8; it was 7a2b3cc3bb3a92ce927e685d33efe8efd75577efbe207b267cb66f68afae7ce9.", "src": "/root/.ansible/tmp/ansible-tmp-1609859402.42-239440084920902/tmpOWkKx8", "url": "https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
```

Reply:
My bad, quick copy paste while I was waiting for the image to be in quay :D